radix-engine-interface/src/api/types/actor.rs

```diff
@@ -1,20 +1,15 @@
-use sbor::rust::string::String;
-use sbor::*;
-
 use crate::api::types::*;
 use crate::model::*;
-use crate::scrypto;
-use crate::Describe;
+use crate::*;
+use sbor::rust::string::String;
 
-#[derive(Debug, Clone, Eq, PartialEq)]
-#[scrypto(Categorize, Encode, Decode)]
+#[derive(Debug, Clone, Eq, PartialEq, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
```

Comment:
Should all of these just default to `ScryptoSbor` unless we have a good reason not to implement Schema? I guess it depends on how it affects compile time.

Perhaps to save wasted work if compile time is awful, we could try cheekily making eg `ScryptoCategorize` also output `ScryptoDescribe` and see how much all the hashing in ScryptoDescribe slows down compilation.

Reply:
I don't think it would save compile time, because the whole macro package is combined into a shared libraray. The only difference is ONE or TWO external function calls.

Reply:
I meant more whether adding the new `Describe` trait to all of the types in the system causes too much slow down in compilation because of its internal uses of `const_sha1` (which can't be very efficient!). Hopefully not :).

I expect we'll want `Describe` on many types anyway so they can form parts of wider schemas (eg the schema for `Instruction`)